components/calendar/index.en-US.md

```diff
@@ -18,6 +18,7 @@ When data is in the form of dates, such as schedules, timetables, prices calenda
 <code src="./demo/notice-calendar.tsx" clientOnly>Notice Calendar</code>
 <code src="./demo/card.tsx" clientOnly>Card</code>
 <code src="./demo/select.tsx" clientOnly>Selectable Calendar</code>
+<code src="./demo/week-number.tsx" clientOnly>Week number</code>
```

Comment:
```suggestion
<code src="./demo/week-number.tsx" clientOnly version="5.23.0">Week number</code>
```

Reply:
The demo that adds the version will have ribbons showing the version.
![image](https://github.com/user-attachments/assets/3e077c87-acca-4c75-8640-4ce999b709d5)